start-all.sh

```diff
@@ -104,6 +104,7 @@ Options:
   --limit container,param        - Allow to set a specific Docker parameter for a container, where container can be:
                                    prometheus, grafana, alertmanager, loki, sidecar, grafanarender
   --archive  data-directory      - Treat data directory as an archive. This disables Prometheus time-to-live (infinite retention), and would run a minimal mode
+  --quick-startup                - If set, the script will not validate that each of the processes start correctly.
```

Comment:
A thought:
start-all could have two modes of operations
1. This "--quick-startup" means start-all doesn't wait at all for anything to come up.
2. The other option is to run each service with --quick-startup, but then wait for all of them together.
The existing mode of operation, starting each background process in order and waiting for each one separately, should probably not exist at all. If you don't think option 1 should become the default, option 2 can. The current situation (wait in order) isn't useful at all. Or so I think.